tests/specs/open-db.js

```diff
@@ -56,6 +56,20 @@
             });
         });
 
+        it('should allow avoidance of server methods', function (done) {
+            var spec = this;
+            db.open({
+                server: 'query',
+                version: 1,
+                noServerMethods: true
+            }).then(function (s) {
+                spec.server = s;
+                expect(spec.server).toBeDefined();
+                expect(spec.server.query).toEqual(jasmine.any(Function));

```

Comment:
I don't think the `expect` really provides much benefit. It'd be better to setup a schema and assert that the `server.<tableName>` doesn't exist.